File: official/docs/python/addresses/create_verified_address.py
import easypost
easypost.api_key = "EASYPOST_API_KEY"

address = easypost.Address.create(
    verify=True,
    street1="417 Montgomery Streat",
    street2="5",
    city="SF",
    state="CA",
    zip="94104",
    country="US",
    company="EasyPost",
    phone="415-123-4567"
)

print(address)

File: official/docs/java/addresses/CreateVerifiedAddress.java
import com.easypost.EasyPost;
import java.util.HashMap;

public class CreateVerifiedAddress {
    public static void main(String[] args) {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
        
        HashMap<String, Object> addressParams = new HashMap<String, Object>();

        addressParams.put("street1", "417 Montgomery Street");
        addressParams.put("street2", "5");
        addressParams.put("city", "SF");
        addressParams.put("state", "CA");
        addressParams.put("zip", "94104");
        addressParams.put("country", "US");
        addressParams.put("company", "EasyPost");
        addressParams.put("phone", "415-123-4567");
        addressParams.put("verify", true);

        Address address = Address.create(addressParams);

        System.out.println(address);
    }
}
